cloud/pkg/cloudstream/tunnelserver.go

```diff
@@ -126,7 +125,18 @@ func (s *TunnelServer) connect(r *restful.Request, w *restful.Response) {
 		sessionID:     hostNameOverride,
 	}
 
-	s.updateNodeKubeletEndpoint(hostNameOverride)
+	err = s.updateNodeKubeletEndpoint(hostNameOverride)
+	if err != nil {
+		klog.Errorf("Failed to Update KubeletEndpoint Port, send reconnect msg to edgeStream")
```

Comment:
No more questions, just one last point.
Unnecessary log. The log has been printed in the `updateNodeKubeletEndpoint ` method.